.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -56,6 +56,7 @@ locals {
 
   artifact_bucket            = try(local.secret["s3_buckets"]["artifact"]["name"], "")
   cellxgene_bucket           = try(local.secret["s3_buckets"]["cellxgene"]["name"], "")
+  datasets_bucket            = try(local.secret["s3_buckets"]["datasets"]["name"], "")
```

Comment:
just for my own clarification--where are all these secrets actually set? Is this secret already set?

Reply:
great question, I was confused on that initially too 😅 

this is set by the happy-env module in the single cell infra; I added the necessary changes [here](https://github.com/chanzuckerberg/single-cell-infra/pull/745/files#diff-d94f492fad3dd0bd4db31186d7558d6079078e53d0b9999af5b69a8947c620adR99-R102)